src/app/services/api.service.ts

```diff
@@ -11,10 +11,10 @@ export class ApiService {
   ) { }
 
   sesion = {
-    client: '3qz2bAjjPkl3yx8D2OD03w',
+    client: 'Vgg-vHyRLKvKkWnnKVM_QQ',
```

Comment:
@germanpacken por favor dar prioridad para mantener esto en "secrets".  

https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets